lib/kanal/plugins/batteries_bridge/bridges/telegram_bridge.rb

```diff
@@ -18,7 +18,7 @@ def setup
               val
             end
 
-            input_convert :tg_button_pressed, :button_pressed do |val|
+            input_convert :tg_button, :button do |val|
```

Comment:
Check Kanal library PR for name of input parameter: https://github.com/idchlife/kanal/pull/27#discussion_r1151644906

Reply:
Renamed